daphne/http_protocol.py

```diff
@@ -230,6 +230,7 @@ def handle_reply(self, message):
         """
         Handles a reply from the client
         """
+
```

Comment:
Avoid adding vertical whitespace just because you think it looks nicer. Restrict your changes to only the part of the code the PR affects.
```suggestion
```